build.yaml

```
@@ -0,0 +1,26 @@
+targets:
+  $default:
+    builders:
+      json_serializable:
+        options:
+          checked: true
+          field_rename: snake
+          any_map: false
+          create_factory: true
+          create_to_json: true
+          disallow_unrecognized_keys: false
+          explicit_to_json: false
+          generic_argument_factories: false
+          ignore_unannotated: false
+          include_if_null: true
+
+      freezed:
+        options:
+          checked: true
+          field_rename: snake
+          ignore_for_file:
+            - type=lintß
```

[nits]
後ろに余計なものがついていそうです👀
```suggestion
            - type=lint
```